pkg/shell/cockpit-server.js

```diff
@@ -393,6 +406,76 @@ PageServer.prototype = {
         $(self.realms).on('notify:Joined.server',
                           $.proxy(self, "update_realms"));
         self.update_realms();
+
+        /* PCP logging
+         */
+
+        /* TODO: Talk to systemd directly.  This is a bit too
+         * cumbersome just for a single service so we cheat and use
+         * cockpitd which has all the hairy code.
+         */
```

Comment:
We can't get change notifications from systemctl.

But you are right, of course.  The main issue is https://bugs.freedesktop.org/show_bug.cgi?id=69575, which makes it non-trivial to listen to Unit states.

I'll scope On/Off out of this PR.